### PR TITLE
Header variant tweaks

### PR DESCRIPTION
**Pull request by zeftilldeath**
Colour changes and making the sizing reflect the sizing of the five words in the main header.

Variant three:
variant_three

Variant four:
variant_four

Variant five:
variant_five

@NataliaLKB Sorry this is probably a right mess.

### CODE REVIEW

**Comment by NataliaLKB** [comment]
After checking out how this looks on some devices (as discussed in person) this looks good 👍

**Comment by paperboyo** [comment]
Are the secondary nav links in variants three and five swipable? If yes: it’s more obvious in variant three (also: the first blur on site).

**Review by NataliaLKB — APPROVED** [review]

**Comment by NataliaLKB** [comment]
@paperboyo 
> Are the secondary nav links in variants three and five swipable?

three is, but five is not

**Comment by paperboyo** [comment]
Anecdotally, @kaelig insisted readers immediately recognised ability to swipe through nav in UX lab. I never believed him for a moment, but used it myself from time to time with a feeling of superiority as an insider who is among two people that know about it... 😜

File: static/src/stylesheets/layout/_ab-new-header-test-variant.scss [hunk 1]
```diff
@@ -823,32 +831,43 @@ $navigation-horizontal-padding: $gs-gutter * 2 + $gs-gutter/2;
...
+    }
+
+    @include mq($from: mobile, $until: mobileLandscape) {
         font-size: 16px;
+        font-size: 5.1vw;
     }
 
     @include mq(mobileLandscape) {
-        font-size: 18px;
+        font-size: 22px;
```
Comment (kaelig) [inline static/src/stylesheets/layout/_ab-new-header-test-variant.scss]: I'm curious how this scales with user text zoom

File: static/src/stylesheets/layout/_ab-new-header-test-variant.scss [hunk 2]
```diff
@@ -755,11 +761,17 @@ $navigation-horizontal-padding: $gs-gutter * 2 + $gs-gutter/2;
             }
         }
     }
+
+    // Single line limited variant, stacked variant
+    &.tertiary__variant-four,
```
Comment (kaelig) [inline static/src/stylesheets/layout/_ab-new-header-test-variant.scss]: Shouldn't variants be modifiers? I haven't seen the full markup but that would seem to make sense.

**Comment by kaelig** [comment]
I got mentioned so I took a peek and left a few comments, feel free to ignore them :)

**Pull request merged**